fe/fe-core/src/main/java/com/starrocks/qe/VariableMgr.java

```diff
@@ -495,7 +495,7 @@ public static List<List<String>> dump(SetType type, SessionVariable sessionVar,
 
                 // For session variables, the flag is VariableMgr.SESSION | VariableMgr.INVISIBLE
                 // For global variables, the flag is VariableMgr.GLOBAL | VariableMgr.INVISIBLE
-                if ((ctx.getFlag() > VariableMgr.INVISIBLE) && !sessionVar.isEnableShowAllVariables()) {
```

Comment:
better to provide comment when sessionVar could be null.

Reply:
done, set the note front the function.